docs/docs/models-definition.md

```diff
@@ -1075,6 +1075,51 @@ User.findAll({ include: [{ model: Tool, as: 'Instruments' }] }).then(function(us
 })
 ```
 
+When eager loading we can also filter the associated model using `where`. This will return all
```

Comment:
Hmm, it appears theres some duplication in models usage and models definition - eager loading should be n usage